index.js

```diff
@@ -20,12 +28,28 @@ const loadYml = _.memoize(function readFile(filePath) {
   }
 });
 
+/**
+ * getSiteConfig
+ *
+ * gets anf.yml configuration file from the site's directory
+ *
+ * @param {Object} site
+ * @returns {Object} JS Object from site's anf.yml file
+ */
 function getSiteConfig(site) {
   const ymlPath = path.resolve(site.dir, 'anf.yml');
 
   return loadYml(ymlPath);
 }
 
+/**
+ * sanitizeComponent
+ *
+ * recursively iterates through a component and its `components` (if any) and removes disallowed properties and unrenderable components from lists
```

Comment:
Long line.

Also, it logs warnings when it finds a component it can't render, right?